README.md

```diff
@@ -24,29 +24,48 @@ You may install the package using npm install command:
 npm install ibm_db
 ```
 
-### Important Environment Variables
+> For more installation details please refer:  [INSTALLAION GUIDE](https://github.com/ibmdb/node-ibm_db/blob/master/INSTALL.md)
+
+
+### Important Environment Variables and Download Essentials 
 
 `IBM_DB_HOME :`
-```
-USE: Set this environment variable if you want to avoid downloading of odbc/clidriver from the IBM provided URL
-     or from the internet.
 
-How : Set `IBM_DB_HOME` environment variable to a pre-installed `db2 client or server installation directory`.
-```
+- USE: Set this environment variable if you want to avoid downloading of clidriver from the [IBM Hosted URL](#downloadCli) or from the internet.
+
+- How: Set **IBM_DB_HOME** environment variable to a pre-installed **db2 client or server installation directory**.
 
 `IBM_DB_INSTALLER_URL :`
-```
-USE: Set this environment variable to by-pass the IBM provided URL for downloading odbc/clidriver.
 
-HOW: Set `IBM_DB_INSTALLER_URL` environment variable with alternate odbc/clidriver downloading URL link
-     or with locally downloaded "tar/zipped clidriver's" parent directory path.
+- USE: Set this environment variable to by-pass the IBM Hosted URL for downloading odbc/clidriver.
 
-TIP: If you don't have alternate hosting URL then, you can download the tar/zipped file of clidriver from
-     IBM hosted URL and can set the downloaded "tar/zipped clidriver's" parent directory path to
-     `IBM_DB_INSTALLER_URL` environment variable. No need to untar/unzip clidriver.
-```
+- HOW: Set **IBM_DB_INSTALLER_URL** environment variable with alternate odbc/clidriver downloading URL link or with locally downloaded
+       "tar/zipped clidriver's parent directory path.
+
+- TIP: If you don't have alternate hosting URL then, you can download the tar/zipped file of clidriver from the [IBM Hosted URL](#downloadCli) and
+       can set the downloaded "tar/zipped clidriver's" parent directory path to **IBM_DB_INSTALLER_URL** environment variable.
```

Comment:
Oh yes, we do set a variable to a PATH, ahhh confusing lines :) I will change that.
and also gonna mention "do not change the file name" :) Thanks for review @sam-github